app.js

```diff
@@ -0,0 +1,245 @@
+// OBJECT AND METHOD
+// FOR PIKE PLACE
+
+var pikePlace = {
+  min: 17,
+  max: 88,
+  cookiesPerHour:[],
+  avgCookiesPerCustomer: 5.2,
+  customersPerHour: function(min, max) {
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+  },
+  generateCookies: function() {
+    for (var i = 0; i < 8; i++){
+      this.cookiesPerHour[i] = Math.round(this.customersPerHour(this.min, this.max) * this.avgCookiesPerCustomer);
+    }
+  }
+}
+
+// Call function to generate cookies
+// for Pike Place
+
+pikePlace.generateCookies();
+
+// Create <li> for
+// Pike Place elements on page
+
+var parent = document.getElementById('Pike-Place');
+var child;
+var total = 0;
+for (i = 0; i < 8; i++){
```

Comment:
This stuff looks great, but in the future you should try to put stuff like this in a function attached to the object itself.